dev/tasks/build-release-bundle

```diff
@@ -51,7 +51,7 @@ echo ${VERSION} > ${BUNDLE_DIR}/version
 CRDS_FILE=$(mktemp -t crds.XXXXXXXX.yaml)
 
 # Update container registry for kustomize
-IMAGE_PREFIX=${IMAGE_PREFIX:-"gcr.io/gke-release/cnrm/"}
+IMAGE_PREFIX=${IMAGE_PREFIX:-"gcr.io/gke-release/"}
```

Comment:
Could you add a comment to tell how this is used and works in the release pipeline? We want to figure out where the problem comes from so we can determine how to avoid it for the future. In other words, we may need some additional checker and validator in the release pipeline to block this change. Before that, we want to know how it is called.

Reply:
Please do not resolve this comment if we haven't had a clear answer yet. But if you do, feel free to add the investigation result  here.